scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -489,4 +490,10 @@ export function ChangeImageController($scope, gettext, notify, modal, _, api, $r
             }
         });
     };
+
+    // init poi if not set
+    if (!$scope.data.poi) {
+        $scope.data.poi = {x: 0.5, y: 0.5};
```

Comment:
@petrjasek @karelpetrak setting poi to 0.5 x 0.5 and saving crops is a potential for users making mistake. If we default the poi to 0.5 for portrait images then the default crops generated will be wrong.  

As per the current media editor change, the user could directly go to metadata tab make changes click "Save" and "Done" button. If we don't enforce users to select the poi then they are bound to make mistakes. This would be fine if system does not need crops.

Reply:
what about setting it only for landspace pictures then and display some warning for portraits that it must be picked?

Reply:
There will still be mistakes with landscape pictures if it has a face on the top/side for [example](https://ichef.bbci.co.uk/images/ic/1200x675/p06rs7fd.jpg).

I agree with @mdhaman - if the client doesn't need crops, let's not crop at all and if he does, let's put validation somewhere. It can be on upload, on publishing or in some other point, but publishing without validation at all doesn't sound good.

Reply:
ok will reuse that validate crop config then and only preserve the default when it's off

Reply:
makes sense?

Reply:
If I understand correctly you want to push default crops for instances where `validatePointOfInterestForImages` is false and keep everything else the same?

```
// init poi if not set
    if (!$scope.data.poi) {
```

Shouldn't this if also have a check on `validatePointOfInterestForImages`? We shouldn't set defaults where `validatePointOfInterestForImages === true`.

When I look back at `validatePointOfInterestForImages` config, it seems that it shouldn't even exist and instead it may be better to have a config called `enableImageCropping` which would ensure that POI is selected and if not set, we wouldn't set POI at all. I don't think we should use default POI.

Reply:
that `data.poi` must be set always otherwise poi editor doesn't work, that was always there but before it was initialized outside the controller using it